mqueue.c

```diff
@@ -573,8 +659,9 @@ int mq_send(mqd_t mqd, const char *ptr, size_t len, unsigned int prio)
             sigev = &mqhdr->mqh_event;
 #if !defined(WIN32)
             if (sigev->sigev_notify == SIGEV_SIGNAL) {
-                sigqueue(mqhdr->mqh_pid, sigev->sigev_signo,
-                                         sigev->sigev_value);
+                /*sigqueue(mqhdr->mqh_pid, sigev->sigev_signo,
+                                         sigev->sigev_value);*/
+                kill(mqhdr->mqh_pid, sigev->sigev_signo);
```

Comment:
I wasn't quite sure what is correct here, can you comment?  So I put back the original code